### PR TITLE
Change cancel button behavior to return to store Settings page

### PR DESCRIPTION
**Pull request by hlhartley**
Change Cancel button behavior to return to the store Settings page

### CODE REVIEW

**Comment by hlhartley** [comment]
[ch7255]

File: app/views/spree/admin/product_feeds/new.html.erb [hunk 1]
```diff
@@ -8,5 +8,5 @@
 
 <%= form_for [:admin, @product_feed], html: { class: "new_product_feed product-feed-form-container" } do |f| %>
   <%= render partial: "form", locals: { f: f, action: "new_product_feed" } %>
-  <%= render partial: "spree/admin/shared/new_resource_links" %>
+    <%= render partial: "spree/admin/shared/edit_resource_links", locals: { collection_url: admin_stores_url } %>
```
Comment (cnorm35) [inline app/views/spree/admin/product_feeds/new.html.erb]: looks like the indentation is off here

Comment (hlhartley) [reply app/views/spree/admin/product_feeds/new.html.erb]: Fixed. Thanks!

**Comment by hlhartley** [comment]
Will squash commits before merging

**Comment by fastjames** [comment]
Could you set up another PR that integrates this into `engine_storefront`? That way we will have a review app for clicking.

**Comment by hlhartley** [comment]
> Could you set up another PR that integrates this into `engine_storefront`? That way we will have a review app for clicking.

Sure! I just created a PR in engine_storefront:
https://github.com/geminimvp/engine_storefront/pull/1312

**Pull request merged**